src/handlers/dns/DnsStreamHandler.h

```diff
@@ -158,6 +161,7 @@ class DnsMetricsBucket final : public visor::AbstractMetricsBucket
     void to_prometheus(std::stringstream &out, Metric::LabelMap add_labels = {}) const override;
     void update_topn_metrics(size_t topn_count) override
     {
+        _dns_topECSQuery.set_topn_count(topn_count);
```

Comment:
Should we change `_dns_topECSQuery` to `_dns_topQueryECS` to line up with the metric name?

Reply:
agreed. done